upi/vsphere/variables.ps1

```diff
@@ -0,0 +1,167 @@
+# Modify these variables below for your environment
+# Option for script to create install-config and modify for upi install
+$createInstallConfig=$false
+$downloadInstaller=$false
+$uploadTemplateOva=$false
+$generateIgnitions=$false
+$waitForComplete=$false
+$delayVMStart=$false
+
+# OKD variables
+# OKD version to be installed
+$version = "4.9"
+$clustername = "openshift"
+$basedomain = "vmc.devcluster.example.com"
+$sshkeypath = "/home/bubba/.ssh/id_rsa.pub"
+# trying to make this as simple as possible
+# will reuse IPIs haproxy, keepalived
+# then we can simply use DHCP.
+$apivip = "192.168.1.10"
+$ingressvip = "192.168.1.11"
+
+
+# vCenter variables
+$vcenter = "vcs8e-vc.ocp2.dev.cluster.com"
+$username = ""
+$password = ''
+$portgroup = "ocp-ci-seg-1"
+$datastore = "workload_share_vcs8eworkload_lrFsW"
+$datacenter = "IBMCloud"
+$cluster = "vcs-8e-workload"
+$vcentercredpath = "secrets/vcenter-creds.xml"
+
+$pullsecret = @"
+{"auths":{"fake":{"auth":"aWQ6cGFzcwo="}}}
+"@
+
+$dns = "8.8.8.8"
+$gateway = "192.168.14.1"
+$netmask = "255.255.255.0"
+
+$lb_ip_address = "192.168.14.10"
+$bootstrap_ip_address = "192.168.14.11"
+
+# The IP addresses to assign to the control plane VMs. The length of this list
+# must match the value of control_plane_count.
+$control_plane_memory = 16384
+$control_plane_num_cpus = 4
+$control_plane_count = 3
+$control_plane_ips = "192.168.14.20", "192.168.14.21", "192.168.14.22"
+$control_plane_hostnames = "control-plane-0", "control-plane-1", "control-plane-2"
+
+# The IP addresses to assign to the compute VMs. The length of this list must
+# match the value of compute_count.
+$compute_memory = 8192
+$compute_num_cpus = 4
+$compute_count = 3
+$compute_ips = "192.168.14.30", "192.168.14.31", "192.168.14.32"
+$compute_hostnames = "compute-0", "compute-1", "compute-2"
+
+# If you do not need the script upload a template, you will need to specify the template here.
+# You can also set this if you want to change the default name of template when its uploaded.
+$vm_template = ""
+
+$failureDomains = @"
+[
+    {
+        // Name of the vSphere data center.
+        "datacenter": "datacenter-2",
+        // Name of the vSphere cluster.
+        "cluster": "vcs-mdcnc-workload-4",
+        // Name of the vSphere data store to use for the VMs.
+        "datastore": "mdcnc-ds-4",
+        // Name of the vSphere network to use for the VMs.
+        "network": "ocp-ci-seg-14",
+        // UUID of the distrubted switch which is hosting the portgroup.  This can derived from the MOB.
+        "distributed_virtual_switch_uuid": "50 05 1b 07 19 2b 0b 0a-eb 90 98 54 1d c5 b5 19"
```

Comment:
I wonder if we could just get the `distributed_virtual_switch_uuid` instead of defining the uuid statically?